src/components/Header/NavBar/Authorization/Authorization.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { NavLink, withRouter } from 'react-router-dom';
+
+import { logOut } from '../../../../actions';
+
+export class Authorization extends Component {
+    rendererLogin = () => {
+      const { handleLogOut, auth } = this.props;
+      if (auth) return <NavLink to="/login" onClick={handleLogOut}>Log out</NavLink>;
+
+      return <NavLink to="/login">Login</NavLink>;
+    }
+
+    render() {
+      return (
+        this.rendererLogin()
```

Comment:
(Minor)

In current of this component contents of the rendererLogin could be placed in `render` method.

```
render() {
  const { handleLogOut, auth } = this.props;
  if (auth) return <NavLink to="/login" onClick={handleLogOut}>Log out</NavLink>;
  return <NavLink to="/login">Login</NavLink>;
}
```
It's quite common pattern to have multiple conditional returns in render. (especially useful when displaying loading, or info, that data is missing)